src/sign/types.ts

```diff
@@ -154,6 +160,14 @@ export const updateProposalTypes = {
   ]
 };
 
+export const flagProposalTypes = {
+  FlagProposal: [
+    { name: 'from', type: 'address' },
+    { name: 'space', type: 'string' },
+    { name: 'proposal', type: 'string' }
+  ]
```

Comment:
can we add `timestamp` param like vote or proposal, I'll add `timestamp` to other actions soon